src/utils.js

```diff
@@ -66,4 +66,29 @@ function matches(textToMatch, node, matcher) {
   }
 }
 
-export {checkHtmlElement, getMessage, matches}
+function deprecate(name, replacementText) {
+  const message = getDepreciationMessage(name, replacementText)
+
+  // Notify user that they are using deprecated functionality.
+  // eslint-disable-next-line no-console
+  console.warn(message)
+}
+
+function getDepreciationMessage(name, replacementText) {
```

Comment:
Does this need to be exposed as a separate function at all? I know it's used in the tests, but I would rather see the full message typed out in the tests than use this function in both spots. Otherwise we never really test the output of this function.

My preference would be to remove this function and inline the message creation in `deprecate`:


```js
function deprecate(name, replacementText) {
  // Notify user that they are using deprecated functionality.
  // eslint-disable-next-line no-console
  console.warn(
    `Warning: ${name} has been deprecated and will be removed in future updates.`,
     replacementText
  )
}

Reply:
Sounds good, I will aim to remove this and simplify the check to just make sure console.warn was called.

Reply:
I will remove it, but will also first try to match the consoles, which shouldn't be too crazy.